CoreFunctionality/src/au/gov/asd/tac/constellation/functionality/compare/CompareGraphPlugin.java

```diff
@@ -420,7 +413,10 @@ protected GraphRecordStore compareGraphs(final String title, final GraphRecordSt
                                 }
                                 break;
                             case "destination":
+                                // Intentionally left blank
+                                break;
```

Comment:
I think omitting the break was intentional since the behaviour of the next case is the same

Reply:
I added the explicit behaviour because in my opinion, the readability increases when explicitly mentioning that the case does nothing and will break. I've changed this back in the most recent commit.

Reply:
I suppose readability is all in the eyes of the beholder. For me, I actually find it more readable to group them together since it suggests they're all have the same behaviour (and less lines of code)